README.md

```diff
@@ -503,7 +503,22 @@ BigQuery would result in incorrect value being written to BigQuery as Flink's Ro
     'dataset' = '<bigquery_dataset_name>',
     'table' = '<bigquery_table_name>');
   ```
-  
+
+### Flink Metrics
+Apache Flink allows collecting metrics internally to better understand the status of jobs and 
+clusters during the development process.
+Each operator in Flink maintains its own set of metrics,
+which are collected by the Task Manager where the operator is running.
+Currently, the Flink-BigQuery Connector supports collection and reporting of the following metrics:
+
+| Metric Name                                        | Metric Description                                                                                                                                                                                                                                   | Supported By (At-least Once/Exactly Once) |
+|----------------------------------------------------|------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------|-------------------------------------------|
+| `numberOfRecordsSeenByWriter`                      | Counter to keep track of the total number of records seen by the writer.                                                                                                                                                                             | At-least Once, Exactly Once               |
+| `numberOfRecordsSeenByWriterSinceCheckpoint `      | Counter to keep track of the number of records seen by the writer since the past checkpoint.                                                                                                                                                         | At-least Once, Exactly Once               |
+| `numberOfRecordsWrittenToBigQuery`                 | Counter to keep track of the number of records successfully appended to BigQuery until now.                                                                                                                                                          | At-least Once, Exactly Once               |
```

Comment:
Please elaborate on the different meanings of `append` in at-least-once and exactly-once.

Reply:
If this means the records actually written (committed) to the BigQuery table, then lets use the word written instead of appended